cve_bin_tool/mismatch_loader.py

```diff
@@ -66,14 +67,46 @@ def setup_sqlite(data_dir: str, db_file: str) -> bool:
     return True
 
 
-def main():
+def setup_args():
     """
-    Run the SQLite loader utility
+    Setup command line arguments
     """
+
     db_path = DISK_LOCATION_DEFAULT / DBNAME
     parent_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
     data_dir = os.path.join(parent_dir, "data")
-    if not setup_sqlite(data_dir, db_path):
+
+    parser = argparse.ArgumentParser(description="mismatch loader")
+    parser.add_argument(
+        "--dir",
+        metavar="DATA_DIR",
+        type=str,
+        default=data_dir,
+        help="Data folder location",
+    )
+    parser.add_argument(
+        "--database",
+        "-db",
+        type=str,
+        default=db_path,
+        help="SQLite DB file (optional - default cve.db).",
```

Comment:
The default value is the full path name of the database and not just the database name. Maybe improve the text to state that it is the full path name for the database.